src/core/internal/gc/impl/conservative/gc.d

```diff
@@ -4208,7 +4220,7 @@ debug (LOGGING)
         }
 
 
-        private void log_free(void *p) nothrow @nogc
+        private void log_free(void *p, size_t size) nothrow @nogc
```

Comment:
`size` unused? I suspect you added some additional checks in your local copy. Add it to the `printf`s here instead?

Reply:
Please see the commit message.

This patch isn't quite right yet (as the failing tests show).

Reply:
> Add it to the `printf`s here instead?

Good idea, done.

> This patch isn't quite right yet (as the failing tests show).

Fixed.